### PR TITLE
ENH: changed CTEST_DROP_METHOD from http to https

### PR DESCRIPTION
**Pull request by seanm**
## PR Checklist
- [ ] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [ ] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [ ] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [ ] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [ ] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

**Comment by seanm** [comment]
@thewtex

**Review by dzenanz — COMMENTED** [review]
Does HTTPS work properly?

**Comment by seanm** [comment]
Grepping for `CTEST_DROP_METHOD` I found `https` elsewhere, so I assume yes, but did not test.

@zackgalbreath should this work?

File: CTestConfig.cmake [hunk 1]
```diff
@@ -1,7 +1,7 @@
 set(CTEST_PROJECT_NAME "ITK")
 set(CTEST_NIGHTLY_START_TIME "1:00:00 UTC")
 
-set(CTEST_DROP_METHOD "http")
+set(CTEST_DROP_METHOD "https")
```
Comment (zackgalbreath) [inline CTestConfig.cmake:4]: Confirming that this line works as intended for `open.cdash.org`

**Review by dzenanz — APPROVED** [review]

**Pull request merged**